tests/python/contrib/test_hexagon/rpc/test_launcher.md

```diff
@@ -26,20 +26,38 @@ Here are the steps that are taken to prepare a runtime on a Hexagon device to te
 - Build TVM library with Hexagon support for host machine.
 - Build TVMRuntime library and C++ RPC server for host machine.
 
-To build these pieces, you can use a cmake command as follow.
+Note: before moving forward make sure to export Clang libraries to `LD_LIBRARY_PATH` and Hexagon toolchain to `HEXAGON_TOOLCHAIN`.
+
+To build these pieces, first build Hexagon API application under `apps/hexagon_api`.
 
 ```bash
-cmake -DUSE_HEXAGON_RPC=ON \
-        -DUSE_ANDROID_TOOLCHAIN=/path/to/android-ndk/build/cmake/android.toolchain.cmake \
+cd apps/hexagon_api
+mkdir build
+cd build
+cmake -DUSE_ANDROID_TOOLCHAIN=/path/to/android-ndk/build/cmake/android.toolchain.cmake \
```

Comment:
Just a personal opinion, but I'd be careful with this style.  It's formatted as `bash` code, and the first few lines look like literal bash code.  So some people (probably me included) will potentially copy this whole block and paste it into a terminal, without reading it carefully.  That's a problem because the following lines aren't _literal_ bash commands, but rather human-readable bash pseudocode.

Reply:
I agree with you. Changed the style, PTAL.